hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -154,7 +154,7 @@ def do_draw(self, data):
 @st.cacheable
 @st.defines_strategy()
 def range_indexes(
-    min_size: int = 0, max_size: int = None
+    min_size: int = 0, max_size: Optional[int] = None
```

Comment:
Same one-line-per-arg style here please, and in fact in all the strategies below :slightly_smiling_face:

Reply:
Unfortunately the line fits the characters limit and black formats it the way it is. However, I added a tailing comma so that when you update the black version to the `20.8b1` it will automatically enforce splitting arguments over multiple lines.

I also did the same for other strategies :smiley:

Reply:
I forgot that Python 3.5 does not allow tailing comma at the end of the line. It already crashed the tests for Python 3.5 on the pipelines. So I guess I have to remove them for compatibility with Python 3.5 :cry: 

At the same time it is probably worth filing this to black issue tracker that black forgets to remove tailing commas with `--target-version=py35` flag. This would made me caught this sooner.

Reply:
Don't worry about it!

Reply:
You are right. It is probably not that important to report because when 3.5 support is dropped it shouldn't affect that many people. And since it is a SyntaxError it can be easily spotted and it is more of a programmer's fault since black does not add those commas by itself.